Dockerfile

```diff
@@ -33,7 +33,7 @@ FROM docker.io/centos:centos8
 
 ARG PKGS_LIST=main-packages-list.txt
 
-COPY ${PKGS_LIST} /tmp/main-packages-list.txt
+COPY ${PKGS_LIST} /tmp/
```

Comment:
If the user supplies a different arg for PKGS_LIST we still want it to end up at /tmp/main-packages-list.txt, so I don't think this change is correct.

Reply:
whoops this was not supposed to be here!